scene/3d/sprite_3d.cpp

```diff
@@ -329,7 +329,7 @@ void SpriteBase3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_item_rect"), &SpriteBase3D::get_item_rect);
 	ClassDB::bind_method(D_METHOD("generate_triangle_mesh"), &SpriteBase3D::generate_triangle_mesh);
 
-	ClassDB::bind_method(D_METHOD("_queue_update"), &SpriteBase3D::_queue_update);
+	ClassDB::bind_method(D_METHOD("_queue_redraw"), &SpriteBase3D::_queue_redraw);
```

Comment:
This binding can actually be removed in favor of `callable_mp()`.

Reply:
I tried to do this before but the problem is this:
![image](https://user-images.githubusercontent.com/66727710/189422364-a52ae7a5-6e2a-4601-bf7a-c7c875944789.png)
I believe something about the way `_queue_redraw`is declared needs to be changed, but I'm not sure what exactly. Passing `&Sprite3D:_draw()`, whose definition comes right before it, doesn't result in any errors.

Reply:
Ah the function is defined in parent class, you need to do `(SpriteBase3D *)this`.

Reply:
Ah, a cast. Ok... Peculiar. Done.